docs/reference/release-notes/7.6.asciidoc

```diff
@@ -3,6 +3,13 @@
 
 Also see <<breaking-changes-7.6,Breaking changes in 7.6>>.
 
+[[breaking-7.6.2]]
+[float]
+=== Breaking changes
+
+Authorization::
+* Creation of child API keys (keys created by existing keys) now requires explicit "no privileges" configuration {pull}53647[#53647], https://www.elastic.co/community/security[CVE-2020-7009]
```

Comment:
It's called "child" keys here. Probably better to use a consistent name. I have no preference over either "derived" or "child" keys.

Reply:
Good catch, thanks!